plugins/newsdownloader.koplugin/main.lua

```diff
@@ -20,6 +20,35 @@ local FILE_EXTENSION = ".html"
 local FEED_SOURCE_SUFFIX = "_rss_tmp.xml"
 local NEWS_DL_DIR, FEED_CONFIG_PATH
 
+
+local function deserializeXML(filename)
+    logger.dbg("NewsDownloader: File to deserialize: ", filename)
+    local f, e = io.open(filename, "r")
+    if f then
+        local xmltext = f:read("*a")
+        f:close()
+        return deserializeXMLString(xmltext)
+    else
+        logger.warn("NewsDownloader: XML file not found", filename, e)
+    end
+end
+
+local function deserializeXMLString(xml_str)
+    -- uses LuaXML https://github.com/manoelcampos/LuaXML
+    -- The MIT License (MIT)
+    -- Copyright (c) 2016 Manoel Campos da Silva Filho
+    require("lib/xml")
+    require("lib/handler")
+
+    --Instantiate the object the states the XML file as a Lua table
+    local xmlhandler = simpleTreeHandler()
```

Comment:
As you'd removed the _-- luacheck: ignore_  from this and '_local xmlparser = xmlParser(xmlhandler)_' line

Travis now fails:
>    plugins/newsdownloader.koplugin/main.lua:151:24: accessing undefined variable 'simpleTreeHandler'
    plugins/newsdownloader.koplugin/main.lua:154:23: accessing undefined variable 'xmlParser'

I'm not sure if you're aware of that?